Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "dfu-core"
-version = "0.4.2"
+version = "0.5.0"
```

Comment:
Please do not touch the version next time. Otherwise I can't use cargo-release and it makes my life harder :sweat_smile: